shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -248,7 +253,7 @@ const styles = Styles.styleSheetCreate(() => ({
     padding: Styles.globalMargins.small,
   },
   extraLineText: {
-    height: 34,
+    height: 36,
```

Comment:
:sheep: noticed this causes the form to move up/down on mobile/desktop when the error shows.